dist/sizzle.js

```diff
@@ -917,6 +934,10 @@ Sizzle.attr = function( elem, name ) {
 				null;
 };
 
+Sizzle.escape = function( sel ) {
+	return (sel + "").replace( rcssescape, fcssescape );
```

Comment:
`CSS.escape` is supported in Chrome and FF, can we degrade instead? Also, we still require spacing for all parentheses :-).

We accept `null` input only for internal proposes right? Can we expose only standartized part of the API and deal with our syntax internally? Since there might be a reason for the "\0" exceptions that we don't know of.

That would be a workaround but if you think it is more useful that way, i'm fine of current exposure. 

Are we planning to expose through jQuery API too? I would suggest to do it.